src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package task;
+
+public class Deadline extends Task {
+    protected String dueTime;
+    private static final int TASK_NAME_INDEX = 0;
+    private static final int DUE_DATE_INDEX = 1;
```

Comment:
Good job on the use of access modifiers.

Reply:
Thank you